Readme.md

```diff
@@ -12,12 +12,22 @@ Play the game of Kamon on the console
 yarn
 ```
 
+Also, you can run make 
+```sh
+make install
+```
```

Comment:
These instructions should replace the existing IMO. Do not fear to delete things from the initial project. It's your project now, so feel free to change it as you wish :slightly_smiling_face:

Reply:
Done